tutorials/blockchains/system-chains/asset-hub/register-local-asset.md

```diff
@@ -0,0 +1,96 @@
+
+3. Now, you need to examine all the registered asset IDs. This step is crucial to ensure that the asset ID you are about to register is unique. Asset IDs are displayed in the **assets** column
+
+      ![Asset IDs on Asset Hub](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-2.webp)
+
+4. Once you have confirmed that the asset ID is unique, click on the **Create** button on the top right corner of the page
+
+      ![Create a new asset](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-3.webp)
+
+5. Fill in the required fields in the **Create Asset** form:
+
+    1. **creator account** - the account to be used for creating this asset and setting up the initial metadata
+    2. **asset name** - the descriptive name of the asset you are registering
+    3. **asset symbol** - the symbol that will be used to represent the asset
+    4. **asset decimals** - the number of decimal places for this token, with a maximum of 20 allowed through the user interface
+    5. **minimum balance** - the minimum balance for the asset. This is specified in the units and decimals as requested
+    6. **asset ID** - the selected id for the asset. This should not match an already-existing asset id
+    7. Click on the **Next** button
+ 
+    ![Create Asset Form](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-4.webp)
+
+6. Choose the accounts for the roles listed below:
+
+    1. **admin account** - the account designated for continuous administration of the token      
+    2. **issuer account** - the account that will be used for issuing this token
+    3. **freezer account** - the account that will be used for performing token freezing operations
+    4. Click on the **Create** button
+
+    ![Admin, Issuer, Freezer accounts](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-5.webp)
+
+7. Click on the **Sign and Submit** button to complete the asset registration process
+
+    ![Sign and Submit](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-6.webp)
+
+## Verify Asset Registration
+
+After completing these steps, the asset will be successfully registered. You can now view your asset listed on the [**Assets**](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/assets){target=\_blank} section of the Polkadot.js Apps interface.
+
+![Asset listed on Polkadot.js Apps](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-7.webp)
+
+!!! note
+    Take into consideration that the **Assets** section’s link may differ depending on the network you are using. For the local environment, the link will be `ws://127.0.0.1:8000/#/assets`.
```

Comment:
This link doesn't make sense, however, you could use this one:

https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A8000#/assets

Or you could just say assets tab of Polkadot.js apps when you enter `ws://127.0.0.1:8000` into Custom Endpoint field.